systemtest/src/main/java/io/strimzi/systemtest/libClient/KafkaClient.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.libClient;
+
+import io.strimzi.systemtest.VertxFactory;
+
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.function.IntPredicate;
+
+public class KafkaClient implements AutoCloseable {
+    private static final Logger LOGGER = LogManager.getLogger(KafkaClient.class);
+    private final Map<String, Vertx> clients = new HashMap<>();
+
+    public KafkaClient() {
+    }
+
+    @Override
+    public void close() {
+        for (Map.Entry<String, Vertx> client : clients.entrySet()) {
+            client.getValue().close();
+        }
+    }
+
+    /**
+     * Send messages to external entrypoint of the cluster with PLAINTEXT security protocol setting
+     * @param topicName topic name
+     * @param namespace kafka namespace
+     * @param clusterName kafka cluster name
+     * @param messageCount message count
+     * @return future with sent message count
+     */
+    public Future<Integer> sendMessages(String topicName, String namespace, String clusterName, int messageCount) {
+        String clientName = "sender-plain-" + clusterName;
+        CompletableFuture<Integer> resultPromise = new CompletableFuture<>();
+        Vertx vertx = VertxFactory.create();
```

Comment:
does it mean that for every send or receive messages method, a new Vert.x instance is created? isn't just one enough for the entire test?

Reply:
Same as above (first comment). I will try to use only one instance for whole test and if everything will be stable I don't see any reason to have X instances for one test. Does it make sense to you? You are vet.x guru in our team :)

Reply:
I am no Vert.x guru for sure! :-) But agree with you, no reasons for many Vert.x instances here.

Reply:
I implemented changes that you suggested and everything works as expected. Code is updated